.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
             name: rust/default
             tag: 1.56.1
         environment:
-            RUSTFLAGS: '-D warnings'
+            RUSTFLAGS: '-Dwarnings'
```

Comment:
Is this a typo, shouldn't be `-D warnings`?

Reply:
I don't believe so. In many of the various blogs/forums I read about this I saw both `-D warnings` and `-Dwarnings` used interchangeably as the `RUSTFLAGS` env variable. But, I've changed it back to its original `-D warnings` since yeah I'd agree it's more obviously not a typo